packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -75,6 +77,8 @@ Below are some examples that might be helpful, but additional methods are availa
 
 <Source code={ChatAdapterStateChangesText} />
 
+The ChatAdapter and the ChatComposite are purpose-built to support single chat thread. Only events, such as 'messagesReceived', from the thread that initialized the ChatAdapter, will be triggered. To accommodate multiple threads usage, consider utilizing a more fundamental service, such as the Chat [StatefulClient](./?path=/docs/statefulclient-overview--page), or directly employing the [ChatClient](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-communication-chat/1.0.0/classes/chatclient.html). This will allow management of all chat threads that the user has joined.
```

Comment:
I feel adding ChatClient (not statefulClient) part is kinda telling customers that "we don't support it well, don't use UI library", but that implantation is actually not bad if you use statefulClient + adapter, and we fully support that when using statefulClient + component

So I guess we can weaken(or remove) the part for directly using ChatClient, and mention developer can use 
1. StatefulClient + Component (like this https://azure.github.io/communication-ui-library/iframe.html?viewMode=docs&id=statefulclient-reacthooks-usepropsfor--page#usepropsfor-chat-app-example)
2. or statefulClient + adapter (pull multi thread info using client then open single chat view using adapter+composite)

Reply:
Agree, highlight our library functions would align better with our objectives.